### ISSUE TITLE
Code Table Request - New Unit of Monetary Value = British Pound Sterling

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6527]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

Record values for a collection with values recorded in Pound Sterling - see https://github.com/ArctosDB/data-migration/issues/1438#issuecomment-1637892184

### Context
_Describe why this new value is necessary and existing values are not._

The collection would like to faithfully record purchase values

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmonetary_units

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

~~British Pound Sterling~~
British pound sterling

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

The British pound sterling (symbol: £; code: GBP; referred to as the pound, pound sterling, or colloquially quid) is the official currency of the United Kingdom and several of its territories. - https://en.wikipedia.org/wiki/Pound_sterling

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

N/A

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

N/A

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

N/A

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

N/A

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

N/A

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

N/A

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [ ] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
@ufarrell

**Comment by ufarrell** [comment]
Thanks, this will be handy!

However, to add complication - its technically this one that I need at the moment: https://en.wikipedia.org/wiki/%C2%A3sd. 

"£sd (occasionally written Lsd, spoken as "pounds, shillings and pence" or pronounced /ɛl.ɛsˈdiː/ ell-ess-DEE) is the popular name for the pre-decimal currencies once common throughout Europe, especially in the British Isles and hence in several countries of the British Empire and subsequently the Commonwealth. The abbreviation originates from the Latin currency denominations librae, solidi, and denarii.[1] In the United Kingdom, these were referred to as pounds, shillings, and pence (pence being the plural of penny)."

It hasn't been used in the UK (or Ireland) since 1971. Any issue with adding a legacy currency? And, if not, any suggestions what it should be called if I put in another request? (£sd with symbols seems like it could be messy?)

**Comment by Jegelewicz** [comment]
I made a new issue - but I'm not sure it will work - see my comment there. I am leaving this one, because it looks like others could use it too!

**Comment by Jegelewicz** [comment]
@ccicero and @Nicole-Ridgwell-NMMNHS gave the thumbs up at today's code table meeting.

**Comment by Jegelewicz** [comment]
Added
